guides/common/modules/proc_removing-host-packages.adoc

```diff
@@ -0,0 +1,9 @@
+[id="Removing-Host-Packages_{context}"]
+= Removing Host Packages
+
+To remove a particular package, click *Remove* from the vertical ellipsis of individual packages.
```

Comment:
This file is missing a short introduction like `You can use {Project} to remove packages from managed hosts.` and `.Procedure` with a ordered list.